modules/howtos/examples/Import.java

```diff
@@ -0,0 +1,314 @@
+          Paths.get("howtos/examples/import.json")),
+        StandardCharsets.UTF_8);
+      
+      for (Object row: JsonArray.fromJson(content)) {
+        JsonObject json = ((JsonObject) row);
+        upsertRow(json.toMap());
+      }
+    }
+    catch (java.io.FileNotFoundException e) {
+      System.out.println("handle FileNotFoundException...");
+    }
+    catch (java.io.IOException e) {
+      System.out.println("handle IOException...");
+    }
+  }
+  // end::importJSON[]
+  
+    // tag::importJSON_batch[]
+    public void importJSON_batch() {
+      System.out.println("importJSON_batch");
+      try {
+        String content  = new String(
+          Files.readAllBytes(
+            Paths.get("howtos/examples/import.json")),
+          StandardCharsets.UTF_8);
+        
+        Flux<MutationResult> results = 
+          Flux.fromIterable(JsonArray.fromJson(content))
+            .map(row -> ((JsonObject) row).toMap())
+            .map(map -> preprocess(map))
+            .flatMap(doc -> reactiveCollection.upsert(doc.getId(), doc.getContent()));
+            
+        results.subscribe(System.out::println);
+        results.blockLast(Duration.of(60, ChronoUnit.SECONDS));
+      }
+      catch (java.io.FileNotFoundException e) {
+        System.out.println("handle FileNotFoundException...");
+      }
+      catch (java.io.IOException e) {
+        System.out.println("handle IOException...");
+      }
+      System.out.println("DONE");
+    }
+    // end::importJSON_batch[]
+  
+  // tag::importJSONL[]
+  public void importJSONL() {
+    try {
+        BufferedReader br =
+          new BufferedReader(
+            new FileReader("howtos/examples/import.jsonl"));
+        String line;
+        while ((line = br.readLine()) != null) {
+          Map<String,Object> row =
+            JsonObject.fromJson(line).toMap();
+            
+          upsertRow(row);
+      }
+    }
+    // ...
+    // tag::repeated[]
+    catch (java.io.FileNotFoundException e) {
+      System.out.println("handle FileNotFoundException...");
+    }
+    catch (java.io.IOException e) {
+      System.out.println("handle IOException...");
+    }
+    // end::repeated[]
+  }
+  // end::importJSONL[]
+  
+  // tag::importJSONL_batch[]
+  public void importJSONL_batch() {
+    System.out.println("importJSONL_batch");
+
+    Flux<String> lines = Flux.using(
+      () -> Files.lines(Paths.get("howtos/examples/import.jsonl")),
+      Flux::fromStream,
+      BaseStream::close
+    );
+    Flux<JsonDocument> docs =
+      lines
+          .map(line -> JsonObject.fromJson(line).toMap())
+          .map(map -> preprocess(map));
+          
+    docs.subscribe(System.out::println);
+    Flux<MutationResult> results = 
+      docs.flatMap(doc -> reactiveCollection.upsert(doc.getId(), doc.getContent()));
+      
+    results.subscribe(System.out::println);
+    results.blockLast(Duration.of(60, ChronoUnit.SECONDS));
+
+    // ...
+    // tag::repeated[]
```

Comment:
Is there a section missing here, or should this tag be removed?

Reply:
believe I've removed this (renamed to ::omit also, which is more generally useful)